src/HttpHeader.cc

```diff
@@ -437,6 +443,7 @@ HttpHeader::parse(const char *header_start, size_t hdrLen, Http::ContentLengthIn
 
             /* Barf on stray CR characters */
             if (memchr(this_line, '\r', field_end - this_line)) {
+                hasBareCr = "bare-CR";
```

Comment:
The `bare CR` term does not use a dash. Let's drop it to better match RFCs and other documentation:

```suggestion
                hasBareCr = "bare CR";
```